src/handlers/helm/K8sHelmDescribeHandler.ts

```diff
@@ -0,0 +1,72 @@
+import * as Joi from 'joi';
+import { get } from 'lodash';
+import { Container } from 'typedi';
+import { ActionHandler, ActionSnapshot } from 'fbl/dist/src/models';
+import { IContext, IActionHandlerMetadata, IDelegatedParameters } from 'fbl/dist/src/interfaces';
+import { K8sHelmService } from '../../services';
+import { FBL_ASSIGN_TO_SCHEMA, FBL_PUSH_TO_SCHEMA, ContextUtil } from 'fbl';
+
+export class K8sHelmDescribeHandler extends ActionHandler {
+    private static metadata = <IActionHandlerMetadata>{
+        id: 'a6s.k8s.helm.describe',
+        aliases: ['k8s.helm.describe', 'helm.describe'],
+    };
+
+    private static schema = Joi.object({
+        name: Joi.string()
+            .min(1)
+            .required(),
+        chart: Joi.object({
+            assignTo: FBL_ASSIGN_TO_SCHEMA,
+            pushTo: FBL_PUSH_TO_SCHEMA,
+        }),
+        objects: Joi.object({
```

Comment:
Same here.

Add `.or('assignTo', 'pushTo')` we should make sure at least one of 2 is specified when `objects` is used